docker/docker-compose.2204.57.yaml

```diff
@@ -13,6 +13,7 @@ services:
     image: swift-certificates:22.04-5.7
     environment:
       - WARN_AS_ERROR_ARG=-Xswiftc -warnings-as-errors
+      - MAX_ALLOCS_ALLOWED_1=0
```

Comment:
Is this right? I'd expect it to be `MAX_ALLOCS_ALLOWED_EMPTY_1`...

Reply:
You are right except that "EMPTY" needs to be lower case. Wondering though why the tests have succeeded... I guess missing an allocation counter number is just a warning and not considered a failure.